src/ralph/backends/http/async_lrs.py

```diff
@@ -164,6 +168,12 @@ async def read(  # pylint: disable=too-many-arguments
         if query is None:
             query = LRSStatementsQuery()
 
+        query.limit = chunk_size
+        logger.warning(
+            "The limit query parameter value is overwritten by the chunk_size "
+            "parameter value."
+        )
```

Comment:
Nice, thanks! Would it be possible to log that warning only when `limit` inside the `query` parameter? WYDT about it?

Reply:
Yup, that's better :smile: